src/EA.Weee.Web/Areas/AatfReturn/Views/CheckYourReturn/Index.cshtml

```diff
@@ -285,25 +310,30 @@
             </div>
             <div class="govuk-grid-column-two-thirds">
                 <div class="govuk-grid-column-two-thirds weee-tonnage-right-aligned-display">
-                        @if (Model.ReportOnDisplayOptions.DisplayNonObligated)
-                        {
-                            <p>
-                                @Model.NonObligatedTonnageTotal
-                            </p>
-                        }
-                        else
-                        {
-                            <p class="not-reported-option">
-                                Not selected for reporting this quarter
-                            </p>
-                        }
+                    @if (Model.ReportOnDisplayOptions.DisplayNonObligated)
+                    {
+                        <p>
+                            @Model.NonObligatedTonnageTotal
+                        </p>
+                    }
+                    else
+                    {
+                        <p class="not-reported-option">
+                            Not selected for reporting this quarter
+                        </p>
+                    }
                 </div>
 
                 <div class="govuk-grid-column-one-third weee-tonnage-right-aligned-display">
                     <p>
                         @if (Model.ReportOnDisplayOptions.DisplayNonObligated)
                         {
-                            @Html.NavigationRouteLink("Change", "Change non obligated", AatfRedirect.NonObligatedRouteName, new { returnId = Model.ReturnId, dcf = false })
+                            @(this.WeeeGds().ActionLinkWithEventTracking("Change",
```

Comment:
The NavigationRouteLink html adds a hidden span with some explanation text for screenreaders, i.e. "Change non obligated" just loaded this up on and the hidden text is This link opens in a new browser window... so isn't correct